src/axelrod_dojo/archetypes/fsm.py

```diff
@@ -140,34 +140,30 @@ def parse_repr(cls, s):
         return cls(num_states, rows, initial_state, initial_action)
 
     def receive_vector(self, vector):
-        """Receives a vector and creates an instance attribute called
-        vector."""
-        self.vector = vector
-
-    def vector_to_instance(self):
-        """Turns the attribute vector in to a FSM player instance.
+        """
+        Read a serialized vector into the set of FSM parameters (less initial
+        state).  Then assign those FSM parameters to this class instance.
 
         The vector has three parts. The first is used to define the next state 
         (for each of the player's states - for each opponents action).
 
         The second part is the player's next moves (for each state - for 
         each opponent's actions).
 
-        Finally, a probability to determine the player's first move."""
-
-        num_states = int((len(self.vector) - 1) / 4)
```

Comment:
Let's use `//` instead of `int` (since we only support Python 3+)

Reply:
This code's been removed I think.